### PR TITLE
Add support for z/OS Large Heap to pipeline build

### PR DESCRIPTION
**Pull request by vsebe**
Add z/OS Large Heap to supported platforms.
Fix build parameters initialization for multi-platforms.

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
Jenkins compile plinux jdk11

**Comment by vsebe** [comment]
@jdekonin please review, thank you!

**Review by jdekonin — APPROVED** [review]
lgtm

**Comment by jdekonin** [comment]
@pshipton for review/merge please.

**Comment by vsebe** [comment]
@pshipton ready for your review, thank you!

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy [hunk 1]
```diff
@@ -103,6 +103,7 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
          's390x_linux_xl_uma' : ['11'],
          's390x_zos'      : ['11'],
          's390x_zos_cm'   : ['11'],
+         's390x_zos_xl'   : ['11'],
```
Comment (pshipton) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:106]: Pls add the xl_cm specs as well.

Comment (vsebe) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy:106]: done

**Comment by pshipton** [comment]
Jenkins compile zlinux jdk8

**Review by pshipton — APPROVED** [review]

**Pull request merged**